AUTHORS.txt

```diff
@@ -322,6 +322,7 @@ memoselyk <memoselyk@gmail.com>
 Michael <michael-k@users.noreply.github.com>
 Michael Aquilina <michaelaquilina@gmail.com>
 Michael E. Karpeles <michael.karpeles@gmail.com>
+Michael Felt <aixtools@felt.demon.nl>
```

Comment:
This file is generated automatically from the git commit history.

You don't have to modify it.